solutions/cses-1682.mdx

```diff
@@ -27,14 +27,14 @@ $1$ $x$ if you're running the DFS on $G$ and $x$ $1$ otherwise.
 
 # Proof
 
-Let's do a proof by contradiction. Assume that $\texttt{can[1][x]}$ is true for
-all vertices $x$ in both $G$ and $G'$, and there exists a pair of vertices
-$u, v$ such that $\texttt{can[u][v]} = \texttt{false}$. Since
-$\texttt{can[1][u]}$ is true in $G'$, then $\texttt{can[u][1]}$ must be true in
-$G$. Additionally, $\texttt{can[1][v]}$ must be true in $G$. So, you can travel
-from $u \rightarrow 1 \rightarrow v$, which contradicts the statement that
-$\texttt{can[u][v]} = \texttt{false}$. Thus, $\texttt{can[u][v]}$ is true for
-all vertices $u, v$.
+We will show that if $\texttt{can[1][x]}$ for $1 \leq x \leq n$ in both $G$ and $G'$,
```

Comment:
don't think it's good to abuse latex
can you explain this in layman terms?

Reply:
Yeah I agree, I just used that because it's the notation used in the first half of the solution.
Should I also rewrite the first part of the solution then?